### PR TITLE
#163519147 Implement like and dislike article functionality

### PR DESCRIPTION
**Pull request by filschristian**
#### What does this PR do?
Have API end-points for liking and disliking an article

#### Description of Task to be completed?
Have the following API end-point working:
`POST: api/v1/articles/:slug/like`
`POST: api/v1/articles/:slug/dislike`

#### How should this be manually tested?

- Pull this branch

- Run `npm run migrate:seed`

- Run `npm run test`

- Run `npm run dev`

- With postman send post requests to those API end-points with the slug of the article you want to like or dislike

#### Any background context you want to provide?
Authentication required for these API end-points.

#### What are the relevant pivotal tracker stories?
#163519147

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: controllers/ArticleController.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import 'dotenv/config';
 import { Op } from 'sequelize';
-import { User, Article, Favorite, Follow, Tag } from '../database/models';
+import {
```
[the remark this answers is not in the document]
Comment (kimotho-njoki) [reply controllers/ArticleController.js:3]: is there a reason why this hound comment has not been resolved?

File: controllers/ArticleController.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+  User, Article, Favorite, Follow, Tag
+} from '../database/models';
```
[the remark this answers is not in the document]
Comment (kimotho-njoki) [reply controllers/ArticleController.js:5]: ???????

File: controllers/ArticleController.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (filschristian) [reply controllers/ArticleController.js:3]: @kimotho-njoki they are resolved but they will still remain here as part of the conversation
image
check this

[the remark this answers is not in the document]
Comment (kimotho-njoki) [reply controllers/ArticleController.js:3]: Is this how the arrangement should be?

**Comment by oesukam** [comment]
@filschristian Can you try to rename the branch because it has more than 3 words.
The branch's name should follow this standard: Short 2-3 words summary about what the branch contains 
Please follow the engineering playbook ->`https://github.com/andela/engineering-playbook/blob/master/5.%20Developing/Conventions/readme.md`

**Pull request closed without merging**